cupyx/scipy/ndimage/measurements.py

```diff
@@ -0,0 +1,199 @@
+import numpy
+
+import cupy
+
+
+def label(input, structure=None, output=None):
+    """Label features in an array
+
+    Args:
+        input (cupy.ndarray): The input array.
+        structure (array_like or None): A structuring element that defines
```

Comment:
It is a little odd that this states "array-like" but a `cupy.ndarray` is not an acceptable array-like.

I see that there is NumPy-based manipulation of the structure in `label` which seems fine as such small array operations should be faster on the NumPy side. However, should a `cupy.ndarray` be allowed as input and it can be transfered to the host internally as needed?

Reply:
Agreed. I will make a `cupy.ndarray` available for `structure`.